tools/CheckDates.scala

```diff
@@ -0,0 +1,184 @@
+import sys.process._
+
+object CheckDates {
+  val usage = """
+
+    This tool compares a copyright date in files with the date of last change in git.
+    If the message of last change match regex "\\[*.\\]" (e.g. '[Tech message]')  it will be ignored and 
+    we will take next commit.
+
+    analyzeDates - just shows if anything is ok
```

Comment:
We know from the tool name `CopyrightDateTool` that we are dealing with dates. Less is more?

```
  analyze - list copyright dates that are out of sync with version history
  update - fix incorrect copyright dates
```